.github/pull_request_template/hotfix.md

```diff
@@ -0,0 +1,27 @@
+# 🚨 Hotfix
+
+## Checklist
+- [ ] Critical issue: #___
+- [ ] Maintainer approval obtained
+- [ ] Branch: `hotfix-[issue]-[desc]`
+- [ ] Title: `<component>: <message> Fix #<issue>`
+- [ ] Minimal, targeted change
+- [ ] Thoroughly tested
+- [ ] Ready for immediate release
+- [ ] CI passing
```

Comment:
I think the CI passing check is unnecessary - the reviewer can see the CI status in the PR itself and the maintainer should (in most cases) not merge unless CI is passing.

Alternatively, we could keep this but mention 'CI is passing on fork before making this PR', to encourage people to use their own fork's workers to verify everything passes before creating a PR to the main repo.